platform/darwin/resources/zh-Hans.lproj/Foundation.stringsdict

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>COORD_DEG_LONG</key>
+	<dict>
+		<key>NSStringLocalizedFormatKey</key>
+		<string>%#@degrees@</string>
```

Comment:
Since Chinese doesn’t need to pluralize any strings, leave this file out so Chinese speakers don’t see English. `COORD_DEG_LONG` etc. are already defined in Localizable.strings.